solutions/swb-reference/src/services/swbProjectService.test.ts

```diff
@@ -0,0 +1,428 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+import {
+  CreateProjectRequest,
+  DeleteProjectRequest,
+  GetProjectRequest,
+  GetProjectsRequest,
+  ListProjectsRequest,
+  ProjectService,
+  UpdateProjectRequest
+} from '@aws/workbench-core-accounts';
+import { ProjectStatus } from '@aws/workbench-core-accounts/lib/constants/projectStatus';
+import { Project } from '@aws/workbench-core-accounts/lib/models/projects/project';
```

Comment:
Are these exported in the account's index file?

Reply:
Updated to export and fixing imports.